packages/grafana-runtime/src/config.ts

```diff
@@ -143,6 +144,7 @@ export class GrafanaBootConfig implements GrafanaConfig {
   rudderstackConfigUrl: undefined;
 
   tokenExpirationDayLimit: undefined;
+  pluginExtensions: Record<string, PluginsExtensionConfig> = {};
```

Comment:
Nit: could we do something like `pluginExtensions: GrafanaConfig['pluginExtensions'] = {};` ?

(Only so we don't need to define it in two places, although I know it's a very basic type)